pkg/server/testserver.go

```diff
@@ -716,6 +717,15 @@ func StartTenant(
 	return pgLAddr, httpLAddr, nil
 }
 
+// HealthHandler simply returns status OK. It may make sense to allow
+// passing ready=1 that checks the liveness of the connection to the KV cluster.
```

Comment:
Why is this a top-level function? Let's inline this or at least unexport (this isn't the KV health handler, so as is this adds confusion).

Why isn't this method a noop? You say it should allow passing `?ready=1` but it looks like you explicitly wrote code that makes that return `StatusBadRequest`.

Reply:
I inlined it. 
It is a noop when pulled as `.../health`. There is nothing to return  besides the OK status. Passing any other arguments (including `.../health?ready=1`) will result in Bad Request currently. If we add support in the future that won't be the case but for now we don't support `?ready=1`. The Bad Request status code is so you don't call `?ready=1` and get OK status. That may give a false impression that it checks the readiness of the SQL server.